jerry-port/default/default-debugger-transport.c

```diff
@@ -13,15 +13,15 @@
  * limitations under the License.
  */
 
-#include "debugger.h"
-#include "jcontext.h"
 #include "jerryscript-port.h"
+#include "jmem.h"
```

Comment:
Do you still need `jmem.h` now?

Reply:
this is because for the handshake part, it is allocating some buffer using JMEM_DEFINE_LOCAL_ARRAY, on other platforms like Zephyr, I just call into malloc, but for the Jerryscript default transport, it might make sense to get the memory from the JrS heap since malloc may not be present.  @zherczeg?